### PR TITLE
Redesign the calendar toolbar

### LINKED ISSUE #5972: Calendar: Top toolbar improvements
Set the date and the total duration of the day as in the design below.

- [x] Label at the top left side of the screen (with chosen date displayed) to maintain left aligned Android app layout
- [x] Daily tracked hours on the right

Design: https://app.zeplin.io/project/5a0d3cd27a75add1a8924f6e/screen/5d8a175b0830e2161df7fcee

### PR DESCRIPTION
**Pull request by daividssilverio**
## What's this?
This redesigns the calendar toolbar according to the new designs o/
- [x] Label at the top left side of the screen (with chosen date displayed) to maintain left aligned Android app layout
- [x] Daily tracked hours on the right

### Relationships
Closes #5972 

## How is it done?
With appbars, framelayout, textviews and a toolbar, take a look.
It adds the reduced elevation toolbar effect that we have from the other toolbars in the app when the view isn't scrolled.
If we don't want that, the `dbbf500` commit needs to be reverted.

### Why not another way?
Feel free to suggest something different.

## Review hints
Review by commit.

## :squid: Permissions
🦑 📆 team

Note: Please fix the squashed commit message part with a typo:
`ma🚧 Simplify MaterialScrollBehaviorListener` to `🚧 Simplify MaterialScrollBehaviorListener`

### CODE REVIEW

**Comment by sofokli1** [comment]
In the designs the toolbar has a different background color from the background of the calendar view. In this PR those look the same.

**Comment by daividssilverio** [comment]
@sofokli1 

> In the designs the toolbar has a different background color from the background of the calendar view. In this PR those look the same.

I didn't see that 🤔 I was following the patterns from the others toolbars in the app.
🤔 🤔 🤔 🤔 and maybe we should have the same toolbar color everywhere? ... I'm not sure about this. The component is different than the other toolbars in the app as well.

**Comment by sofokli1** [comment]
keep it as it is for now and we can see later. this isn't a big deal

**Comment by daividssilverio** [comment]
@sofokli1 , I guess it's just a matter of the elevation we see in the designs.
The toolbar color and the background are the same: `#FFFAFBFC`

**Comment by sofokli1** [comment]
In this [design](https://app.zeplin.io/project/5a0d3cd27a75add1a8924f6e/screen/5d8a175b0830e2161df7fcee) I see the tool bar with color #fafbfc and the calendar background with color #ffffff . As i said this is not urgent. This can be easily fixed later on, if even needed at all

File: Toggl.Droid/Resources/menu/CalendarFragmentMenu.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+    <item
```
Comment (semanticer) [inline Toggl.Droid/Resources/menu/CalendarFragmentMenu.xml]: I know that we have this in the design, but maybe we should have a dedicated issue for the transition to toolbar settings?

Comment (daividssilverio) [reply Toggl.Droid/Resources/menu/CalendarFragmentMenu.xml]: There's are issues for all settings screens #5931
Do you want me to 🔥 this?

**Comment by daividssilverio** [comment]
Hey @semanticer , I can't replicate the crash you've found, but it looks very unrelated to this PR's changes. I'll create an issue for it, but if that's your only concern besides https://github.com/toggl/mobileapp/pull/5983#discussion_r329940167 ?

**Comment by pdrj** [comment]
When I was testing other calendar changes, if I didn't clean the solution before building, I got a similar crash to @semanticer. Maybe you can try doing that.

**Comment by daividssilverio** [comment]
Thing is, if this is caused by a dirty build, there's nothing to fix. This looks like a problem with linking the .net libraries for the `ImmutableList`. @semanticer if you clean the solution before build, does it still crash?

File: Toggl.Droid/Resources/menu/CalendarFragmentMenu.xml (hunk 1 above)
Comment (semanticer) [reply Toggl.Droid/Resources/menu/CalendarFragmentMenu.xml]: Up to you, I guess... I can live with this being prepared.

Comment (daividssilverio) [reply Toggl.Droid/Resources/menu/CalendarFragmentMenu.xml]: I'll leave it then.

**Comment by daividssilverio** [comment]
I've created an issue to keep an eye on the reported crash: #6004.

**Pull request merged**